lib/OpenQA/WebAPI/Controller/API/V1/JobGroup.pm

```diff
@@ -187,6 +187,30 @@ sub _validate_common_properties ($self) {
     $validation->optional('description');
 }
 
+sub _check_keep_logs_and_results ($self, $properties) {
```

Comment:
This is still not checking the case when only one of the values is passed. Then we need to compare it to the value already stored in the database.
e.g. if the form  / post parameters only contains `keep_logs_in_days`, we need to get `keep_results_in_days` from the database